### PR TITLE
[Tezos] Remove not used baker update code

### PR DESCRIPTION
**Pull request by hewigovens**
we have been directly calling baker api in blockatlas

### CODE REVIEW

**Review by vikmeup — APPROVED** [review]
@catenocrypt can you review this too?

**Review by catenocrypt — APPROVED** [review]

File: script/blockchain/tezos.ts [hunk 1]
```diff
@@ -28,53 +25,6 @@ function getChainValidatorsList(chain: string): ValidatorModel[] {
     return JSON.parse(readFileSync(`${(getChainValidatorsPath(chain))}/${validatorsList}`));
 }
```
Comment (vikmeup) [inline script/blockchain/tezos.ts:26]: 'getChainValidatorsList' is defined but never used

File: script/blockchain/tezos.ts [hunk 2]
```diff
@@ -1,14 +1,11 @@
...
     validatorsList,
     getChainValidatorsPath,
-    getChainValidatorsListPath,
     getChainValidatorsAssets
 } from "../generic/repo-structure";
 import { Tezos } from "../generic/blockchains";
 import { readFileSync } from "../generic/filesystem";
-import { writeJsonFile } from "../generic/json";
 import { ActionInterface, CheckStepInterface } from "../generic/interface";
 import { ValidatorModel } from "../generic/validator-models";
```
Comment (vikmeup) [inline script/blockchain/tezos.ts:11]: 'BakingBadBaker' is defined but never used

Comment (hewigovens) [reply script/blockchain/tezos.ts:11]: thanks, removed

File: script/blockchain/tezos.ts (hunk 1 above)
Comment (hewigovens) [reply script/blockchain/tezos.ts:14]: @catenocrypt it's safe to remove? this one seems like a generic one might be used later

Comment (vikmeup) [reply script/blockchain/tezos.ts:14]: feel free to remove, we no need to run any logic for validators here, it's manually updated

**Pull request merged**